quarkus.ls/com.redhat.quarkus.ls/pom.xml

```diff
@@ -111,6 +111,11 @@
 			<artifactId>org.eclipse.lsp4j.jsonrpc</artifactId>
 			<version>${lsp4j.version}</version>
 		</dependency>
+		<dependency>
+		    <groupId>org.apache.commons</groupId>
+		    <artifactId>commons-text</artifactId>
+		    <version>1.8</version>
+		</dependency>		
```

Comment:
mmm we're adding 700KB (200KB of commons-text + 500KB of commons-lang) just for that feature. Seems overweight.
We should try to use proper shading to remove unnecessary cruft.

Reply:
LevenshteinDistance from commons-text has none dependencies (just implement an interface but we can skip it), so if we can copy/paste this class, what do you think?

Reply:
LevenshteinDistance class have none dependency (just one for implementing EditDistance interface) so I think we can copy/paste this class.

Reply:
@fbricon I do a copy/paste of LevenshteinDistance  to remove the commons-text dependency.